CITi-UFPE/react-native-boilerplate
# feat(2023-updates)

## What I did

- Added expo-router over react-navigation
- Added react-native-svg
- Updated dependencies
- Setup theme and contexts

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout <branch name>`
   1. `git pull origin <branch name>`
   1. `yarn install`
   1. `code .`
   1. `yarn start`

1. In your browser go to http://localhost:3000

1. Check if the component looks just like the mockup in deferent screen sizes 

1. In VScode open `<relative path>`

1. Things to change in the code during the test
   ```tsx
   if (isAwesome){
     return <Component />
   }
   ```

1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description 

&nbsp;
***
